easybuild/easyconfigs/b/Bazel/Bazel-6.3.1-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,39 @@
+name = 'Bazel'
+version = '6.3.1'
+
+homepage = 'https://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably.
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/bazelbuild/%(namelower)s/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
+patches = [
+    'Bazel-6.3.1_cc-info.patch',
+    'Bazel-6.3.1_add-symlinks-in-runfiles.patch',
+    'Bazel-6.3.1_use_std=c++17_instead_of_0x.patch',
+    'Bazel-6.3.1_fix_missing_cstdin_in_abseil.patch',
+]
+checksums = [
+    {'bazel-6.3.1-dist.zip': '2676319e86c5aeab142dccd42434364a33aa330a091c13562b7de87a10e68775'},
+    {'Bazel-6.3.1_cc-info.patch': 'f0b43ba05c2353a0324f0c7e532e9192f82f6937f68fb3e24fabcca7996ff6aa'},
+    {'Bazel-6.3.1_add-symlinks-in-runfiles.patch': '81db53aa87229557480b6f719c99a0f1af9c69dfec12185451e520b0128c3ae2'},
+    {'Bazel-6.3.1_use_std=c++17_instead_of_0x.patch':
+     'ffb7ab652c962bf437440db72f796c523e9ac768b26e832362dcea3d2ad7e6ef'},
+    {'Bazel-6.3.1_fix_missing_cstdin_in_abseil.patch':
+     'bdb3b1a31f3101de21958018ad3b08a645ef3b96f5d5440a15cba99c2c42c70f'},
```

Comment:
@tanmoy1989 it's complaining about this checksum

Reply:
@verdurin Not sure what to do? I checked my build and they got built fine without any complain. That's what I got when I did: `eb --inject-checksums=sha256 --force`. Any suggestions, please?

Reply:
Just tried it, it's still the same!